### ISSUE TITLE
skip "select header row" and "match columns" stage and directly move to "validate data" stage

### ISSUE THREAD

**Issue by SubhasmitaLMG** [UgnisSoftware/react-spreadsheet-import#226]
@masiulis Can I skip "select header row" and "match columns" stage and directly move to "validate data" stage?
Means "file upload" and  "validate data" stage should be there only.
Can I do that?

**Comment by masiulis** [comment]
Hey @SubhasmitaLMG seems like a duplicate of #188. It will be possible once that one is done. As always PRs are welcome!

**Comment by rishabhjindalcred** [comment]
@masiulis Could you please provide an example for the above? I am having difficulty understanding how to use the 'skip headers' feature.
